diff --git a/Configurations/15-android.conf b/Configurations/15-android.conf
index 7b496a4..d899848 100644
--- a/Configurations/15-android.conf
+++ b/Configurations/15-android.conf
@@ -120,10 +120,10 @@
             } elsif ($user{CC} eq "clang") {
                 die "no NDK clang on \$PATH";
             } else {
-                if (which("$triarch-gcc") !~ m|^$ndk/.*/prebuilt/([^/]+)/|) {
-                    die "no NDK $triarch-gcc on \$PATH";
-                }
-                $cflags .= " -mandroid";
+                # if (which("$triarch-gcc") !~ m|^$ndk/.*/prebuilt/([^/]+)/|) {
+                #     die "no NDK $triarch-gcc on \$PATH";
+                # }
+                # $cflags .= " -mandroid";
                 $user{CROSS_COMPILE} = "$triarch-";
             }
 
